app/src/components/pages/Home.tsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
+import uviclogo from 'images/uviclogo.png';
+import { Typography } from '@mui/material';
+import { Divider, Grid } from '@mui/material';
+import { Container } from '@mui/system';
+import { Box } from '@mui/system';
+import { Button } from '@mui/material';
 
 function Home() {
-  return <div>Home</div>;
+  return (
+    <Container>
+      <Grid container paddingTop={4} spacing={2} direction={'row'}>
+        <Grid item>
+          <img src={uviclogo} alt="uviclogo" width={150}></img>
+        </Grid>
+
+        <Grid item marginTop={1} marginLeft={23}>

```

Comment:
Nit: There's probably a better way to center this text than with `marginLeft={23}`

Reply:
@Hexuan-Z you can use this for reference, as Sam said you don't want to use hardcoded values for alignment
https://stackoverflow.com/questions/50766693/how-to-center-a-component-in-mui-and-make-it-responsive